pkg/cmd/get/get_build_logs_test.go

```diff
@@ -63,6 +63,32 @@ func TestGetTektonLogsForRunningBuild(t *testing.T) {
 	}
 }
 
+func TestGetTektonLogsForRunningBuildWithPendingPod(t *testing.T) {
+	commonOpts := opts.NewCommonOptionsWithFactory(fake.NewFakeFactory())
+	commonOpts.BatchMode = true
+	testCaseDir := path.Join("test_data", "get_build_logs", "tekton_build_logs_pending")
+
+	activities := tekton_helpers_test.AssertLoadSinglePipelineActivity(t, testCaseDir)
```

Comment:
IMO `AssertLoadSinglePipelineActivity` is a bad name here. Really this is about loading the pipeline activity `LoadSinglePipelineActivity`